packages/plugins/plugin-assistant/src/stories/Chat.stories.tsx

```diff
@@ -355,3 +357,101 @@ export const WithSearch = {
     components: [ChatContainer, [GraphContainer, LoggingContainer]],
   },
 } satisfies Story;
+
+export const WithTranscription = {
+  decorators: getDecorators({
+    plugins: [TranscriptionPlugin()],
+    config: config.remote,
+    types: [Transcript.Transcript],
+    onInit: async ({ space, binder }) => {
+      const queue = space.queues.create();
+      const transcription: DataType.Message[] = [
```

Comment:
factor out test data object or generator to ./testing
`const messages = [...].map(({ sender, text }) => Obj.make())`